sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -440,6 +440,13 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(Integer.MAX_VALUE)
 
+  val DRIVER_TIMEZONE = conf("spark.rapids.driver.user.timezone")
+    .doc("This is to set driver timezone. Note that if the user provides this config " +
```

Comment:
As written, this implies it could be used to set the timezone on the driver.  It would be clearer to state that the config is used to inform the executor plugin about the driver's timezone and is not intended to be set by the user.